resolwe_bio/processes/workflows/wgs.yml

```diff
@@ -0,0 +1,474 @@
+- slug: workflow-wgs
+  name: Whole genome sequencing (WGS) analysis
+  data_name: "{{ reads|sample_name|default('?') }}"
+  requirements:
+    expression-engine: jinja
+  version: 1.0.0
+  type: data:workflow:wgs
+  category: Pipeline
+  description: |
+    Whole genome sequencing pipeline analyzes paired-end whole genome sequencing data. It consists of trimming,
+    aligning, marking of duplicates, Picard metrics, recalibration of base quality scores and finally, calling of
+    variants.
+
+    The tools used are Trimmomatic which performs trimming. Aligning is performed using BWA (mem). Marking of
+    duplicates (MarkDuplicates), Picard metrics (AlignmentSummaryMetrics, CollectWgsMetrics and InsertSizeMetrics),
+    recalibration of base quality scores (ApplyBQSR) and calling of variants (HaplotypeCaller) is done using GATK4
+    bundle of bioinformatics tools.
+    Result is a file of called variants (VCF).
+  input:
+    - name: reads
+      label: Raw untrimmed reads
+      type: data:reads:fastq:paired
+      required: true
```

Comment:
`required: true` is the default, so you can omit this line (also below where applicable)